lib/buildsystems/autotools.rb

```diff
@@ -19,7 +19,7 @@ def self.build
       end
       abort 'configure script not found!'.lightred unless File.file?('configure')
       FileUtils.chmod('+x', 'configure')
-      if system('grep -q /usr/bin/file configure')
```

Comment:
The problem here is that our custom system command defined in `lib/package.rb` will retry on failure, where here we actually don't want it do do that. @supechicken, any ideas on how to access the ruby `system` here? Maybe `Kernel.system()`?

I've got plans to refactor that system code out into `commands/build.rb` to stop leakages like this, but that won't happen until most other commands are refactored, and isn't a timely solution to this problem.

Reply:
This is not simple to Rubyize.  If someone has a better solution, I'm all ears.
```
     search = false
      Dir['**/*'].grep(/config$/).each do |filename|
        File.open(filename) do |file|
          file.each_line do |line|
            if line =~ /\/usr\/bin\/file/  
              search = true
              break
            end
          end
        end
      end
      if search
        puts 'Using filefix.'.orange
        system 'filefix'
      end
```
It does no harm to run filefix on every config file even if `/usr/bin/file` doesn't exist.  The filefix program itself can do the checking and fixing in one go.

Reply:
No, I'm simply saying is there a way to run the default `system()` command rather than the one from `lib/package.rb`

Reply:
What package did you encounter the bug on? I'd like to test `Kernel.system()`.

Reply:
I was trying to update a2ps.rb.

Reply:
For what its worth, I'm working on that as well: https://github.com/Zopolis4/chromebrew/tree/0a. Might be best to avoid doubling up. I should PR my branches sooner :sweat_smile:.

Reply:
Tested, using `Kernel.system()` works.

Reply:
> Tested, using `Kernel.system()` works.

Yeah I noticed that too. I mentioned it to @uberhacker and as I understand it he's going to update the PR accordingly.

Reply:
We should probably figure out though why `system()` is broken though. 😅

Reply:
As I said, its because we define a custom `system()` in `lib/package.rb` that restarts with `make -j1` on errors instead of failing.

Reply:
Yes. We should fix the custom `system()`...

That's the root cause of this issue.

Reply:
The custom `system()` is working as intended, that's what its supposed to do. Although, it might be nice to find a way to isolate that to package files only, rather than having to use `Kernel.system()` to override it.